### ISSUE TITLE
No document shown in preview feature, no error thrown

### ISSUE THREAD

**Issue by DrLoiseau** [Myriad-Dreamin/tinymist#758]
Hey, 
I can't see my document when executing tinymist preview file.typ. The navigator (tested firefox and konqueror) only shows a blank page. However there is no error thrown by tinymist. I really don't understand what's happening, any help would be appreciated.

tinymist version : 

tinymist 
Build Timestamp:     2024-10-19T17:10:46.837035115Z
Build Git Describe:  v0.12.0
Commit SHA:          4437b3bffb5f01f06bd64e4e424480ba72a88619
Commit Date:         None
Commit Branch:       None
Cargo Target Triple: x86_64-unknown-linux-gnu
Typst Version:       0.12.0
Typst Source:        git+https://github.com/Myriad-Dreamin/typst.git?tag=tinymist-v0.12.0#58426a90a7ef721738a01be09793d33e55eb75a9

tinymist preview log : 
tinymist preview ~/Boulot/2024/BPH_1/at/06_encephale/eleves_at06_encephale.typ
[2024-11-01T17:07:00Z INFO  tinymist::tool::preview] Arguments: PreviewCliArgs {
        preview: PreviewArgs {
            enable_partial_rendering: false,
            invert_colors: "never",
            task_id: "default_preview",
            refresh_style: OnType,
        },
        compile: CompileOnceArgs {
            input: Some(
                "/home/drloiseau/Boulot/2024/BPH_1/at/06_encephale/eleves_at06_encephale.typ",
            ),
            root: None,
            inputs: [],
            font: CompileFontArgs {
                font_paths: [],
                ignore_system_fonts: false,
            },
            creation_timestamp: None,
            certification: None,
        },
        preview_mode: Document,
        data_plane_host: "127.0.0.1:23625",
        control_plane_host: "127.0.0.1:23626",
        static_file_host: "",
        not_as_primary: false,
        dont_open_in_browser: false,
    }
[2024-11-01T17:07:00Z INFO  typst_preview] Previewer: typst actor spawned
[2024-11-01T17:07:00Z DEBUG typst_preview::actor::typst] TypstActor: waiting for message
[2024-11-01T17:07:00Z INFO  typst_preview] Previewer: editor actor spawned
[2024-11-01T17:07:00Z INFO  tinymist::tool::preview] preview server listening on http://127.0.0.1:23626
[2024-11-01T17:07:00Z INFO  tinymist::tool::preview] Control panel server listening on: 127.0.0.1:23626
[2024-11-01T17:07:00Z INFO  tinymist::tool::preview] preview server listening on http://127.0.0.1:23625
[2024-11-01T17:07:00Z INFO  tinymist::tool::preview] Data plane server listening on: 127.0.0.1:23625
[2024-11-01T17:07:00Z INFO  tinymist::tool::preview] Static file server listening on: 127.0.0.1:23625
[2024-11-01T17:07:01Z INFO  tinymist::task::cache] CacheEvictTask: evict cache in 216.045µs
[2024-11-01T17:07:01Z INFO  tinymist::task::cache] CacheEvictTask: evict cache in 210.705µs

**Comment by Enter-tainer** [comment]
from the log(if that's all of it), it seems that your browser doesnt connect to tinymist. are you using vscode-remote or something similar?

**Comment by DrLoiseau** [comment]
Yes that's the whole log, after connecting to the local address in a navigator. For this example I launched tinymist preview from a terminal, otherwise I use emacs with typst-preview.el 

Is it possible that I miss some dependancy ? typst-preview works well with typst 0.11

**Comment by Enter-tainer** [comment]
Quick question, which address did you open in browser? I'm not sure but i think only one of them is meant to be opened in browser. The other one is for editor features like synctex.

**Comment by DrLoiseau** [comment]
I tried both, but I think the right one is http://127.0.0.1:23625/. It is the one that opens automatically.

**Comment by DrLoiseau** [comment]
Hi,

the problem has been solved after a recent update to my debian packages. I don't know which one was too old, maybe libwebkit2gtk ?
